daemon/src/maker_inc_connections.rs

```diff
@@ -457,28 +457,28 @@ impl Actor {
 ///
 /// Both IO operations, upgrading to noise and checking the version are gated by a timeout.
 async fn upgrade(
-    mut stream: TcpStream,
+    stream: TcpStream,
     noise_priv_key: x25519_dalek::StaticSecret,
     this: xtra::Address<Actor>,
 ) -> Result<()> {
+    let socket_timeout = Duration::from_secs(10);
```

Comment:
❓ Any reason why this value has changed from 20 to 10 seconds?

Reply:
Copy-paste error :)